packages/node_modules/cerebral/package.json

```diff
@@ -22,7 +22,7 @@
     "function-tree": "^1.0.0-beta.3"
   },
   "scripts": {
-    "test": "mocha --compilers js:babel-register \"src/**/*.test.js\" && npm run test_ts",
+    "test": "mocha --compilers js:babel-register \"src/**/*.test.js\" --require src/views/test/dom.js && npm run test_ts",
```

Comment:
This will conflict on rebase after the TypeScript PR. We will have a single "setup" script in "test/setup.js". On rebase we have to move "dom.js content" to this file.